### DISCUSSION TITLE
[Mapping] Fixing HOP's offices standardization

### DISCUSSION THREAD

**Discussion by Fr-Dae** [unitystation/unitystation#8323] [Ideas]
Hello, in order to improve the quality of life and standardize the office from one map to another, 

here is an image representing a simple functional office.

![HOP-Office](https://user-images.githubusercontent.com/23202421/152616126-48e58a7e-f904-425f-b578-b94d130e583c.gif)
![HOP-Office0](https://user-images.githubusercontent.com/23202421/152616129-d8e5767c-da0c-47cc-a89e-986fa50a32be.png)

Basically, all cards should be checked, from the oldest to the most recent. 
thanks you

**Comment by PerfectTangent** [comment]
converting to a discussion.

**Comment by mmails** [comment]
seems highly unnecessary

**Comment by AtnerNT** [comment]
I personally don't see the need to standardize HoP's office, but if we do I have a few grievances with the suggested layout. I want to preface this that im not having a go - this is just my take on the design:
1) The supply console, I dont see why HoP in particualr should be forced to have a supply console, if command is going to have a supply console- which I dont think is always the best of ideas, it should either be in the bridge itslef- accessable to all heads, or in captains office, where only he can mess with it. I don't see why HoP needs one.
2) The labeler? Same with the supply console- I dont see why HoP needs this- maybe in the commands meeting/conference room or the main bridge- But labellers are more for departments that are creating unique goods, like chemist or toxins, or distributing packages like cargo. To me it doesnt make sense why the HoP would have one- especially since custom ID items are already renamed to the current thing when theyve been edited in the HoP console.
3) While I agree that a telescopic baton can sometimes be a good idea- I know on a few maps heads get them, to deal with insubordinates within there ranks and assist sec- I think a taser is overkill, the HoP is like any other head, and I dont think giving CE or CMO tasers is a good idea either. Heads aren't security and they shouldnt be given security gear- its the same reason they get the telebaton and not a stun baton. Captain may be the execption because the captain is expected to be on call for whatever scenario arrives, including security dramas. But not HoP
4) Folders and paper are always a nice idea for any head, but I dont think they need there own personal photocopier. It can be nice, but i dont think its something that should be standardized, maybe one in command accessable to all heads could be nice- so all the heads of departments can come copy important paper work at command, but I really dont think they need there own one, especially on smaller maps like Fall, Mini and Square
5) I dont think all HoP's offices need double shutters- but I do think its agood idea that if you do have 2 sets of shutters- you should always have two buttons aswell. My main complaint here is that I think mandatory arrows on maps is a bit overkill and isnt really needed, and the trouble makers that ruin the lines anyways will most likely ignore it

All that being said- I would like to see other peoples take on the matter

**Reply by Fr-Dae** [reply]
> don't see the need to standardize HoP's office

 **it's just extremely annoying,** between two parties to never have the same thing in the locker because nobody has defined a list of items that should be in it.
**It completely changes the game experience.**

moreover it's just an amalgamation of what is ALREADY offered in the game, but standardized.

==============

### **1.** The HOP is a civil and bureaucratic officer.
He is the segond of the captain, the person in charge of the cargo (he has to report to the QM), manages the services, and keeps the station active.

He has the same rank as the HOS, except that he is essentially in charge of security.

normally the QM (if there is one) must be able to provide a paper with what is entering and leaving the station (more or less detailed) 
since the reports are not written automatically by the console.
The QM is often "lazy" to do this, but it is within his remit.
Therefore the HOP must be able to check that the cargo ship is not doing anything with the station's background. because the officers cannot be antag. and the HOP must be able to inform the captain of the station's situation, who "technically" must be able to command from his office.
**already present in some of the HOP offices**

==========================
### **2.** The labelers are considered as office equipment in the game. 
and that's what it is, you will see more often a Dymo in an office to organize files, folders than elsewhere. (in any case used more frequently) 
there are regular cases in game where the labeler helps in RP, and solves problems. 
it remains almost as important as paper and pencil.
**already present in some of the HOP's offices**

==========================
### **3.** once again you misunderstand the role and rank of the HOP, 
all as the HOS is an officer in segond, (or senior officer)

the taser is already provided in some offices, and it's not bad, because if we are attacked, the taser (which I remind you is not lethal) is a guarantee of safety. at least the time that the security comes to handcuff the person. the whole brig (including the HOP office) is considered a code 309 - Major Trespass 
and taking any object in the office is considered a 408 - Grand Theft. 
we are not here to joke. it is not because it is a bureaucratic job that we should think it is useless, 
after the captain, the HOP is the direct representative of the bureaucratic directives of Nanotrasen
after that it is a V42 or a V43, I do not have a preference, the V43 just allows to shoot through a window, and the V42 has two extra shots. 
and I said it above.  **already present in some of the HOP's offices**

===========================
### **4.** The HOP is, with the laywer and curator, the only role that uses the photocopier.
When the characters parade in front of the desk, sometimes with strange requests and/or incomprehensible accents, we literally have no time to eat...
Regarding the size, I proved with the image above that a FULLY functional office only requires 3x5 (+ 2x6 for the glass) as a first officer, I can't see myself begging for access to the copy machine.
and no the captain does not use the copier, he is not even required to know how it works. 
**already present in some of the HOP's offices**

===================================
### **5.** The dual security panel, controlled by two separate buttons, is a standard feature on most maps.
It allows only one person to be processed at a time.
It also ensures that they are not assaulted by surprise, or robbed of whatever is on the desk in front of them (by shoving them).
moreover it allows to have the office open without being directly accessible. 

the arrows are essentially there to increase the bureaucratic immersion, as in "paper Please" except that here it is "Id Please". 
It is not necessarily useful to have a long corridor.
3 spaces are enough to separate the current case from the people waiting 
**already present in some of the HOP's offices**

**Reply by Fr-Dae** [reply]
### **exemple  of a low size possibility 9x5tiles**
![HOP-Office2](https://user-images.githubusercontent.com/23202421/152655371-a693585b-fdc9-46bc-991a-8813288ff403.png)

**Reply by mmails** [reply]
The HoP isn't in charge of cargo, the QM is.

The HoP is in charge of all civil services on station. (Janitor, Cook, Lawyer and whatnot)

**Reply by Fr-Dae** [reply]
Actually he is, since he is the captain's mate, and directly responsible for the services AND the QM.
this is for the moment badly written in the wiki but on the TG Wiki it is clearly stated as such

the QM is a sector leader
the HOS and HOS are officers.

**Reply by Fr-Dae** [reply]
![EmbeddedImage](https://user-images.githubusercontent.com/23202421/174051210-4617ad1b-6599-4ac6-b0db-45528abecbc9.png)
and this is not the subject of the conversation. here we ask for the standardization of all HOP offices with the same model, we are not in a debate lore. but mapping. 
Please keep your personal considerations in mind,

**Reply by mmails** [reply]
Okay, but right now there are bigger problems, bugs, important features to add/fix

**Reply by Fr-Dae** [reply]
bugs are the problem of the devs,
this request is for MAPPERS, so what's the connection?
